html/modules/custom/docstore/src/Controller/DocumentController.php

```diff
@@ -584,18 +584,22 @@ protected function createDocumentFromParameters(NodeType $node_type, array $para
           $uuid = $file['media_uuid'];
         }
 
+        // Existing media.
+        if (!empty($uuid) && !$this->validateEntityUuid('media', $uuid)) {
+          $media = $this->loadMedia($uuid);
+        }
+        // Existing file.
+        elseif (!empty($uuid) && !$this->validateEntityUuid('file', $uuid)) {
```

Comment:
@attiks I don't think the file entity uuids are ever exposed in the API (only the media entity uuids). Is there any use case for this?

Related to this, the line below doesn't seem correct: `$media = $this->loadFile($uuid);`. Shouldn't it be the media associated with the file entity that should be loaded rather than the file entity?